docs/Quickstart.md

```diff
@@ -15,6 +15,10 @@ Make sure that you have received a **developer APP token**, **API token** and **
 
 ## Installation
 
+### Prepare git-lfs
```

Comment:
Does it take a lot of time to download those dependencies?

Reply:
Yes it does, the ios-webrtc is 900mb for example

Reply:
Perhaps specify then it might take a long time to install.